### ISSUE TITLE
Boot keyboard is not able to enter BIOS

### ISSUE THREAD

**Issue by tomaszduda23** [adafruit/Adafruit_TinyUSB_Arduino#208]
### Operating System

Linux

### IDE version

5.2.0

### Board

lolin_s2_mini

### ArduinoCore version

2.0.5

### TinyUSB Library version

1.14.4

### Sketch (attached txt file)

[main.txt](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/files/9758988/main.txt)

### What happened ?

Unable to enter BIOS without reseting board during boot-up. It can be fixed by doing detach and attach or HW reset.
```
  TinyUSBDevice.detach();
  delay(100);
  TinyUSBDevice.attach();  
```

### How to reproduce ?

1. Power on boot keyboard
2. Try to enter BIOS

### Debug Log

[boot_usb_kbd.txt](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/files/9758974/boot_usb_kbd.txt)

```
USBD Xfer Complete on EP 80 with 0 bytes
  Queue EP 83 with 8 bytes ...                          <- should there be complete for this EP?

USBD Setup Received 80 06 00 01 00 00 12 00 
  Get Descriptor Device
  Queue EP 80 with 18 bytes ...
USBD Xfer Complete on EP 80 with 18 bytes
```

**Comment by hathach** [comment]
try to remove Serial connetion and leave only keyboard interface alone to see if that works.

**Comment by hathach** [comment]
closed due to lack of user response
